Outdated code in plotutil/main.go, doesn't compile

<!--
Please make sure your issue title matchs the Go convention; a summary
of the problem, prefixed by the primary affected package.

If the issue is a question about usage or similar, please ask at the
Gonum discussion list rather than filing an issue:

https://groups.google.com/forum/#!forum/gonum-dev
-->
### What are you trying to do?

Run the plotutil sample program

### What did you do?
<!-- 
Please include a link to a minimal reproducer here. Preferably this
should be a complete runnable example at https://play.golang.org/.

If the issue involves a plot and can be replicated in an image.Image
rendering, render the plot using pic.ShowImage(img) from the tour
package, "golang.org/x/tour/pic".
-->
```
$ cd plotutil
$ go run main.go 
```

### What did you expect to happen?
<!-- Please include an expected output image if available. -->
Generate a plot

### What actually happened?
<!-- Please include an annotated output image if available. -->

```
# command-line-arguments
./main.go:89:11: assignment mismatch: 2 variables but plot.New returns 1 values
./main.go:128:9: assignment mismatch: 2 variables but plot.New returns 1 values
```

### What version of Go and Gonum/plot are you using?
<!--
Paste the output of `go version` and if you are installing Gonum/plot from source, paste
the output of `(cd $(go env GOPATH)/src/gonum.org/v1/plot && git rev-parse HEAD)`.
-->
```
$ go version
go version go1.16 linux/amd64
$ git rev-parse HEAD
caab7788e04dfd7a74cec4ee80f55b7d1f9a2bfa
```
This is Gonum v0.9.0

### Does this issue reproduce with the current master?

Yes

--- a/plotutil/main.go
+++ b/plotutil/main.go
@@ -86,10 +86,7 @@ func Example_errpoints() *plot.Plot {
 		}
 	}
 
-	plt, err := plot.New()
-	if err != nil {
-		panic(err)
-	}
+	plt := plot.New()
 
 	mean95, err := plotutil.NewErrorPoints(plotutil.MeanAndConf95, pts...)
 	if err != nil {
@@ -125,10 +122,7 @@ func (n stackValues) Value(i int) float64 {
 
 // An example of making a stacked area chart.
 func Example_stackedAreaChart() *plot.Plot {
-	p, err := plot.New()
-	if err != nil {
-		panic(err)
-	}
+	p := plot.New()
 
 	p.Title.Text = "Example: Software Version Comparison"
 	p.X.Label.Text = "Date"
@@ -147,7 +141,7 @@ func Example_stackedAreaChart() *plot.Plot {
 		{0, 0, 0, 0, 0, 0.68, 5.67},
 	}
 
-	err = plotutil.AddStackedAreaPlots(p, plotter.Values{2007, 2008, 2009, 2010, 2011, 2012, 2013},
+	err := plotutil.AddStackedAreaPlots(p, plotter.Values{2007, 2008, 2009, 2010, 2011, 2012, 2013},
 		"Version 3.0",
 		stackValues{vs: vals[0:7]},
 		"Version 2.1",

